### DISCUSSION TITLE
Statistics

### DISCUSSION THREAD

**Discussion by filipemr** [Unipisa/Simu5G#73] [Q&A]
I'm using simu5g in a emulated scenario. But always the drop and loss related statistics are 0 or NaN. I ran some simulation examples too and there is the same behavior. Is this a simulator problem? Or is there some config to enable this statistic to be collected?

**Comment by giovanninardini** [comment]
Losses depend on the configured scenario. If, for instance, you emulate/simulate only one UE very close to its serving base station it is likely that packets are not going to be lost.
